app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -238,6 +59,7 @@ export default class DEditor extends Component {
   @service("emoji-store") emojiStore;
   @service modal;
 
+  editorComponent = TextareaEditor;
```

Comment:
This is the single entrypoint that defines that this DEditor is working with the textarea implementation